homeassistant/components/linkplay/config_flow.py

```diff
@@ -0,0 +1,20 @@
+"""Config flow to configure LinkPlay component."""
+
+from linkplay.controller import LinkPlayController
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_entry_flow
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+
+async def _async_has_devices(hass: HomeAssistant) -> bool:
+    """Return if there are devices that can be discovered."""
+    session = async_get_clientsession(hass)
+    controller = LinkPlayController(session)
+    await controller.discover_bridges()
```

Comment:
This does not seem to be able to discover a Wiim Pro that I have here for testing.

Reply:
Which is another reason relying on zeroconf will be helpful.

Reply:
Devices are now discovered through zeroconf